data/data_model.md

```diff
@@ -11,7 +11,8 @@ Please do not include any information beyond that specified in the format below.
 | version        | String         | Required version string for the data model. **MUST** be set to **`2.0`**  |
 | id             | String         | Required ID for this event. **MUST** be in UUIDv4 format |
 | start_time     | [Timestamp](https://tools.ietf.org/html/rfc3339#page-10)      | Time the activity started, in UTC. **MUST** be in [RFC 3339, section 5.6](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) **date-time** format. For example: `2018-11-13T20:20:39+00:00` | 
-| tid     | Array | Array of techniques, including subtechniques that were observed. One or many techniques can be accepted|
+| technique     | Array | Array of techniques, including subtechniques that were observed. One or many techniques can be accepted|
+| tactic     | String | \*\***Only if available\*\*** The tactic to which the technique belongs, under the context of this detection. For example, if `Scheduled Jobs` was detected as a means to get system access, this would be `Privilege Escalation`, and not `Persistence`.|
```

Comment:
What if there are multiple techniques that are under different tactics? It's not clear to me how you would use this.